src/workers/parser/utils/ast.js

```diff
@@ -14,7 +14,20 @@ function _parse(code, opts) {
     code,
     Object.assign({}, opts, {
       sourceType: "module",
-      plugins: ["jsx", "flow", "objectRestSpread"]
+      plugins: [
+        "estree",
```

Comment:
Turns out this plugin was the problem @jasonLaster removing it reverted a lot of the snapshot changes.